plume/src/main/kotlin/io/github/plume/oss/passes/type/GlobalTypePass.kt

```diff
@@ -28,7 +29,13 @@ class GlobalTypePass(private val driver: IDriver) : ITypePass {
      *     NAMESPACE_BLOCK(<global>) -(AST)-> TYPE_DECL
      *     FILE(<unknown>) -(CONTAINS)-> TYPE_DECL
      *     FILE(<unknown>) <-(SOURCE_FILE)- TYPE_DECL
+     *
+     *     TYPE_DECL -(REF)-> TYPE
+     *     TYPE_DECL -(AST)-> *MEMBER ? String[].length ?
```

Comment:
I'll make issues out of these - I forgot that primitives need bindings, members, and method stubs